src/libpriv/rpmostree-kernel.c

```diff
@@ -303,6 +318,23 @@ rpmostree_run_dracut (int     rootfs_dfd,
   glnx_fd_close int tmp_fd = -1;
   g_autofree char *tmpfile_path = NULL;
   g_autoptr(RpmOstreeBwrap) bwrap = NULL;
+  g_autoptr(GPtrArray) rebuild_argv = NULL;
+
+  g_assert (argv != NULL || rebuild_from_initramfs != NULL);
+
+  if (rebuild_from_initramfs)
+    {
+      rebuild_argv = g_ptr_array_new ();
+      g_ptr_array_add (rebuild_argv, "--rebuild");
```

Comment:
This tricked me for a bit. The `--rebuild` option doesn't show up in the manpage for `dracut`.